tests/test.sh

```diff
@@ -203,12 +11,10 @@ ossl()
 
 ############################## MAIN ##############################
 
-setup
-
-title PARA "Export Public key to a file"
-ossl 'pkey -in $BASEURI -pubin -pubout -out ${TSTCRT}.pub'
-title LINE "Export Public key to a file (pub-uri)"
-ossl 'pkey -in $PUBURI -pubin -pubout -out ${TSTCRT}.pub'
+#title PARA "Export Public key to a file"
+#ossl 'pkey -in $BASEURI -pubin -pubout -out ${TSTCRT}.pub'
+#title LINE "Export Public key to a file (pub-uri)"
+#ossl 'pkey -in $PUBURI -pubin -pubout -out ${TSTCRT}.pub'
 title LINE "Export Public key to a file (with pin)"
```

Comment:
Why are these commented out now ?

Reply:
I was debugging something and forgot to clean this up and it was hidden in the move from one file to the other. Sorry.